server/services/healthCheck.test.ts

```diff
@@ -36,22 +36,18 @@ describe('Healthcheck', () => {
 
 function successfulCheck(name: string): HealthCheckService {
   return () =>
-    new Promise((resolve, _reject) => {
-      resolve({
-        name: `${name}`,
-        status: 'ok',
-        message: 'some message',
-      })
+    Promise.resolve({
+      name: `${name}`,
+      status: 'ok',
+      message: 'some message',
     })
 }
 
 function erroredCheck(name: string): HealthCheckService {
   return () =>
-    new Promise((resolve, _reject) => {
-      resolve({
-        name: `${name}`,
-        status: 'ERROR',
-        message: 'some error',
-      })
+    Promise.resolve({
+      name: `${name}`,
```

Comment:
surprised it couldn't work out that either `name: name` or simply `name` would work here

Reply:
oh good spot! you should have waited to merge and i'd have changed that too :)